### DISCUSSION TITLE
Suggestion: find more people to develop this system together

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#355] [Ideas]
It is recommended that you find more people to compile and maintain this project, especially the system. It is impossible for one person to have multiple roles (although this has been mentioned before). If a volunteer is willing to help, you can ask him to fix minor problems first.  Or happy to design icons and system themes or UI to make the system more beautiful, so that you have a second-hand that can come in handy when you are busy or beyond your ability.

**Comment by probonopd** [comment]
Please see https://hellosystem.github.io/docs/developer/contributing - how can we get more interested developers to find and test helloSystem?

**Comment by louies0623** [comment]
A description like "How you can help this project" should be added to the welcome notification window in hello.
Demo
![fbfhgm 1](https://user-images.githubusercontent.com/44593430/162103884-73d64536-86c4-498c-a053-9748e3a1fe7b.png)

**Reply by probonopd** [reply]
Love it. Can you make it happen?

Code for the Welcome Wizard is at https://github.com/helloSystem/Utilities/blob/master/System/Welcome.app/Resources/welcome-wizard.py - it's very short and it's Python so it should be relatively easy to understand :)

**Reply by louies0623** [reply]
I can copy the text to you, but its content is translated by google, so it is not very accurate. By the way, is there an emoji in hello?

**Reply by louies0623** [reply]
@probonopd I can't do it, the coding ability is beyond me

**Comment by louies0623** [comment]
Text:
" 
Thank you very much for using the Hello system, Since this system is under development, we need more  people to participate, so your assistance can make  this system more complete.

How can help us? you can

🖥 Use the Hardware Probe in the system to submit hardware information.
💬 Submit a problem report or feature suggestion on Hello.
🏞 Submit a beautiful wallpaper (Please review submission specifications).
🇺🇳Submit language translations to reach more places.
🖍 Submit system UI and theme idea or sketch design, Make the user experience better.
🛠 Participate in developing systems and software compilation to increase the stability.
🎙 Promote on social media to let more people know.
❤ Donate.

We would greatly appreciate your assistance. 🙌
"
